src/core_plugins/timelion/public/panels/timechart/schema.js

```diff
@@ -168,15 +168,15 @@ module.exports = function timechartFn(Private, config, $rootScope, timefilter, $
 
         let legendScope = $scope.$new();
         function drawPlot(plotConfig) {
+          if (!$('.chart-canvas', $elem).length) $elem.html(template);
+          const canvasElem = $('.chart-canvas', $elem);
 
-          if (!plotConfig || !plotConfig.length) {
-            $elem.empty();
```

Comment:
this case no longer needs to be handled ? what would happen now if plotConfig is empty ?